domain/miningmanager/miningmanager.go

```diff
@@ -21,6 +21,7 @@ type MiningManager interface {
 	ValidateAndInsertTransaction(transaction *externalapi.DomainTransaction, isHighPriority bool, allowOrphan bool) (
 		acceptedTransactions []*externalapi.DomainTransaction, err error)
 	RevalidateHighPriorityTransactions() (validTransactions []*externalapi.DomainTransaction, err error)
+	RemoveTransactions(txs []*externalapi.DomainTransaction, removeRedeemers bool) error
```

Comment:
Seems like this change slipped in here (which is why factory.go seems to have a compilation error). 

Also, please make sure `go fmt` is not required for miningmanager.go, the tests seem to fail on something there.